### ISSUE TITLE
duplicate classes on #refs div starting in Pandoc 2.14

### ISSUE THREAD

**Issue by jjallaire** [jgm/pandoc#7370]
When rendering an HTML document using a bibliography and citeproc, the #refs `<div>` is being adorned with duplicate classes for `references csl-bib-body hanging-indent` starting in Pandoc 2.14. Using the examples files below, the following command:

```bash
pandoc test.md --to html --bibliography=bibliography.bib --citeproc
```

**test.md**

```markdown
---
title: Hello World
---

## This is a test document

This is some text @knuth1984

::: {#refs}
:::
```

**bibliography.bib**

```latex
@article{knuth1984,
  title={Literate programming},
  author={Knuth, Donald E.},
  journal={The Computer Journal},
  volume={27},
  number={2},
  pages={97--111},
  year={1984},
  publisher={British Computer Society}
}
```

Yields the following output for each version of pandoc:

**2.13**

```html
<h2 id="this-is-a-test-document">This is a test document</h2>
<p>This is some text <span class="citation" data-cites="knuth1984">Knuth (1984)</span></p>
<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
  <div id="ref-knuth1984" class="csl-entry" role="doc-biblioentry">
Knuth, Donald E. 1984. <span>“Literate Programming.”</span> <em>The Computer Journal</em> 27 (2): 97–111.
</div>
</div>
```

**2.14**

```html
<h2 id="this-is-a-test-document">This is a test document</h2>
<p>This is some text <span class="citation" data-cites="knuth1984">Knuth (1984)</span></p>
<div id="refs" class="references csl-bib-body hanging-indent references csl-bib-body hanging-indent" role="doc-bibliography">
  <div id="ref-knuth1984" class="csl-entry" role="doc-biblioentry">
Knuth, Donald E. 1984. <span>“Literate Programming.”</span> <em>The Computer Journal</em> 27 (2): 97–111.
</div>
</div>
```

**Comment by jgm** [comment]
I think I already fixed this in 76e5f047b0ef4c293687f9ddce62d601b23058a9
Try a nightly.

**Comment by jjallaire** [comment]
Great, thanks!
